.gitignore

```diff
@@ -1,17 +1,20 @@
-*~
-*.pyc
-*.pyo
-*.swp
-parser.out
-parsetab.py
-.idea
+/.cache
 /.tox
+/.tox-*
+/.venv
+/.venv-*
 /build
 /dist
 /MANIFEST
-/stone.egg-info
 /setuptools-3.1-py2.7.egg
 /setuptools-3.1.zip
-/.cache
-.DS_Store
+/stone.egg-info
 .arcconfig
+.DS_Store
+.idea
+parser.out
+parsetab.py
+*.pyc
+*.pyo
+*.swp
+*~
```

Comment:
Haha interesting. I'd always go least-specific to most. So I guess you're likely to prepend to your list, and I'm likely to append.